README.rst

```diff
@@ -864,6 +864,8 @@ Choose from the list of available rules:
     ``get_defined_constants``. User constants are not accounted in this list
     and must be specified in the include one; defaults to ``true``
   - ``include`` (``array``): list of additional constants to fix; defaults to ``[]``
+  - ``scope`` (``'all'``, ``'namespaced'``): only fix constant invocations that are made
```

Comment:
The order of options is `all, namespaced`, however the description order is `namespaced, all`. Could be confusing.

Reply:
that's exactly the same wording than for the `scope` option of the `native_function_invocation` fixer.

Reply:
Fix that one too? 😇 I dunno, I'll leave that call to (one of) the maintainers.

Reply:
I wasn't comparing anything to anything, I just ment in this specific line in itself. It reads `a, b: description of b, or a.`

Reply:
Im not saying that you were, still no change needed IMHO